webapp/publisher/views.py

```diff
@@ -15,6 +16,11 @@
 )
 
 
+account_page = flask.Blueprint(
```

Comment:
Any reason not to call it account, rather then account_page?

Reply:
@Lukewh the problem here is that I already have some vars with this name, I think it might be less confusing by having a specific name for the blueprint (we have the same var name for the store and the blog blueprints

Reply:
Does it not then make more sense for the other vars to be more specific? Account is quite a general term for single vars whereas `account_page` is, I'd say, too specific for a bucket of sub-resources.

Another reason is to match the name of the blueprint with the endpoint, having a different name could cause confusion